openapi-generator/templates/java/libraries/google-api-client/build.gradle.mustache

```diff
@@ -24,7 +24,7 @@ if(hasProperty('target') && target == 'android') {
 
     apply plugin: 'com.android.library'
     apply plugin: 'com.github.dcendents.android-maven'
-
+    
```

Comment:
nit but should these added whitespaces be here?

Reply:
hm that change has been reverted, make sure to check the latest state of the PR